azurekeyvault-flexvolume/main.go

```diff
@@ -154,6 +157,9 @@ func Validate(options Option) error {
 		if options.podNamespace == "" {
 			return fmt.Errorf("-podNamespace is not set")
 		}
+		if options.nmiPort == "" {
```

Comment:
we should also validate the value provided and error handle bad port input.

Reply:
I have added validation for port input, checking for integer and within valid port range